extensions/amp-lightbox-gallery/1.0/amp-lightbox-gallery.js

```diff
@@ -81,6 +84,18 @@ class AmpLightboxGallery extends BaseElement {
     this.removeAsContainer?.();
   }
 
+  /** @override */
+  onViewGrid() {
+    super.onViewGrid();
+    triggerAnalyticsEvent(this.element, 'thumbnailsViewToggled', dict({}));
+  }
+
+  /** @override */
+  onToggleCaption() {
+    super.onToggleCaption();
```

Comment:
Is the toggleCaption button only clickable when it is being expanded to cover the overflow? Can it also perform a "minimize" action, in which case the analytics event would be sent out at the wrong time

Reply:
It can perform the minimize and the same analytics event should fire in that case as well. 0.1 for reference:
https://github.com/ampproject/amphtml/blob/b855a57dd3db4fd51e6036a1b4ad8de469c51db7/extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js#L453-L459

Docs for reference:
https://amp.dev/documentation/components/amp-lightbox-gallery/#descriptionoverflowtoggled

Reply:
Thank you for the link